### DISCUSSION TITLE
Release 0.4.0

### DISCUSSION THREAD

**Discussion by JakaKokosar** [biolab/orange3-survival-analysis#46] [General]
## What's Changed
* [FIX] owkaplanmeier: fix median line with multiple KMplot widgets on the canvas by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/24
* [FIX] owkaplanmeier: hide legend on empty input by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/25
* [ENH] Survival tables, CoxRegression model and learner by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/27
* [FIX] Various bug fixes and improvements by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/28
* [ENH] Detect and omit missing rows by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/29
* [ENH] owcohorts: add new widget by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/30
* [ENH] Set valid types for survival and group variables in DomainModel by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/31
* New icons by @larazupan in https://github.com/biolab/orange3-survival-analysis/pull/32
* [ENH] move survival annotations to attributes of columns by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/33
* New As Survival Data icon by @larazupan in https://github.com/biolab/orange3-survival-analysis/pull/35
* [FIX] owassurvivaldata: fix variable selection on widget startup by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/34
* owcohorts: if only 2 unique values found don't split on whole array by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/36
* [ENH] owranksurvivalfeatures: minor UI improvements by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/37
* [ENH] owstepwisecoxregression: minor UI changes by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/38
* [ENH] owkaplanmeier: If input domain contains variable 'Cohorts' autoselect it by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/39
* [FIX] owkaplanmeier: fix crashing legend by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/41
* [ENH] owkaplanmeier: add annotated data output by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/43
* [ENH] Simplify conversion of Table to DataFrame by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/44
* [ENH] use new interface for testing and scoring learners by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/42
* [FIX] owcohorts: transform domain before computing by @JakaKokosar in https://github.com/biolab/orange3-survival-analysis/pull/45

**Full Changelog**: https://github.com/biolab/orange3-survival-analysis/compare/0.3.0...0.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/biolab/orange3-survival-analysis/releases/tag/0.4.0'>Release 0.4.0</a>.</em>

**Comment by GiSteps** [comment]
Hi, the current version of the Kaplan-Meier widget is not working for me (the previous one was working well) (seems it is not able to identify the groups). Could you resolve the problem?
